### ISSUE TITLE
Wrong mapping while branch "develop" has been exist.

### ISSUE THREAD

**Issue by nhahv** [OpherV/gitflow4idea#100]
In my case, i created a branch named "develop" then init the git-flow. 
But the prefix configuration came like this on `.git/confing` file.

```
[gitflow "branch"]
    master = master
    develop = develop
[gitflow "prefix"]
    feature = develop
    release = feature/
    hotfix = release/
    support = hotfix/
    versiontag = support/
```

btw, when i init git-flow by bash terminal, everything work fine.

**Comment by Zl0n1k** [comment]
I have the same issue.

**Comment by OpherV** [comment]
Thanks - I'll look into this!

**Comment by robertegnacheski** [comment]
I get the same issue was well.

**Comment by OpherV** [comment]
Committed a fix for the upcoming version

**Comment by OpherV** [comment]
Fixed in 0.6.1
